sequence/tests/test_sequence_manager.py

```diff
@@ -11,69 +12,80 @@ def setUp(self):
         self.id_generator = IdGenerator()
 
     @mock.patch.object(sequence.dynamo_sequence.DynamoSequenceGenerator, 'next')
-    async def test_generate_message_id(self, mock_next):
-        mock_next.return_value = 1
+    @async_test
+    async def test_generate_transaction_id(self, mock_next):
+        mock_next.return_value = awaitable(1)
         message_id = await self.id_generator.generate_transaction_id()
-        mock_next.assert_called_with(key='transaction_id')
-        self.assertEquals(message_id, 1)
+        mock_next.assert_called_with('transaction_id')
+        self.assertEqual(message_id, 1)
 
     @mock.patch.object(sequence.dynamo_sequence.DynamoSequenceGenerator, 'next')
+    @async_test
     async def test_generate_interchange_id(self, mock_next):
-        mock_next.return_value = 5
+        mock_next.return_value = awaitable(5)
         interchange_id = await self.id_generator.generate_interchange_id('sender', 'recipient')
-        mock_next.assert_called_with(key='SIS-sender-recipient')
-        self.assertEquals(interchange_id, 5)
+        mock_next.assert_called_with('SIS-sender-recipient')
+        self.assertEqual(interchange_id, 5)
 
     @mock.patch.object(sequence.dynamo_sequence.DynamoSequenceGenerator, 'next')
+    @async_test
     async def test_generate_message_id(self, mock_next):
-        mock_next.return_value = 9
+        mock_next.return_value = awaitable(9)
         message_id = await self.id_generator.generate_message_id('sender', 'recipient')
-        mock_next.assert_called_with(key='SMS-sender-recipient')
-        self.assertEquals(message_id, 9)
+        mock_next.assert_called_with('SMS-sender-recipient')
+        self.assertEqual(message_id, 9)
 
     @mock.patch.object(sequence.dynamo_sequence.DynamoSequenceGenerator, 'next')
+    @async_test
     async def test_generate_interchange_id_for_empty_sender(self, mock_next):
         with self.assertRaises(ValueError):
             mock_next.return_value = 5
```

Comment:
shouldn't mock_next.return_value be awaitable for all instance of this? Its an async method

Reply:
done